yt_dlp/extractor/vtm.py

```diff
@@ -1,60 +1,42 @@
-from .common import InfoExtractor
-from ..utils import (
-    int_or_none,
-    parse_iso8601,
-    try_get,
-)
+from .medialaan import MedialaanBaseIE
+from ..utils import str_or_none
+from ..utils.traversal import require, traverse_obj
 
 
-class VTMIE(InfoExtractor):
-    _WORKING = False
-    _VALID_URL = r'https?://(?:www\.)?vtm\.be/([^/?&#]+)~v(?P<id>[0-9a-f]{8}(?:-[0-9a-f]{4}){3}-[0-9a-f]{12})'
-    _TEST = {
+class VTMIE(MedialaanBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?vtm\.be/[^/?#]+~v(?P<id>[\da-f-]+)'
```

Comment:
I like having the actual UUID pattern in here
```suggestion
    _VALID_URL = r'https?://(?:www\.)?vtm\.be/[^/?#]+~v(?P<id>[\da-f]{8}(?:-[\da-f]{4}){3}-[\da-f]{12})'
```